nav-app/src/app/matrix/matrix-flat/matrix-flat.component.scss

```diff
@@ -50,6 +50,11 @@
                         font-size: 16px;
                         line-height: 12px;
                     }
+                    &.disabled {
+                        background: #aaaaaa;
+                        border-color: #aaaaaa;
+                        box-shadow: none;
+                    }
```

Comment:
I think we also should add `cursor: default` (or `&:not(.disabled)` to the line making the cursor pointer) since currently the cursor still communicates that the element is clickable.